### PR TITLE
Add screenshot method to env

### PR DESCRIPTION
**Pull request by spdskatr**
Modify Dockerfile to fix the location of the Xauthority file, add a
method to call a screenshot tool from inside the docker container and
then output it to the output volume

### CODE REVIEW

File: gym_sts/envs/base.py [hunk 1]
```diff
@@ -8,16 +8,20 @@
 from typing import Optional, Tuple, Union
 
 import docker
-import gym
 from docker.models.containers import Container
 
+import gym
```
Comment (kronion) [inline gym_sts/envs/base.py]: This isn't the right section for this import to be in -- it should be in the third-part import section along with `docker`. I'm guessing that you didn't make this change, and that pre-commit changed it automatically. Can you double check that you are in your virtualenv (e.g. `poetry shell` if you're using poetry)? If you're not, pre-commit can mistakenly classify libraries as first-party.

**Review by kronion — CHANGES_REQUESTED** [review]
Just one lint-related issue, otherwise LGTM.

File: gym_sts/envs/base.py (hunk 1 above)
Comment (spdskatr) [reply gym_sts/envs/base.py]: I did run the pre-commit hooks inside `poetry shell`... digging deeper, when I run `isort` manually, it tells me `else-type place_module for gym returned FIRSTPARTY` which is strange. Still have not found a fix for it, any idea what the problem might be?

Comment (spdskatr) [reply gym_sts/envs/base.py]: Fixed isort issue on my end and fixed import order

**Review by kronion — APPROVED** [review]

**Pull request merged**